specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -106,6 +106,63 @@
             }
           }
         }
+      },
+      "patch": {
+        "tags": [
+          "BillingAccounts"
+        ],
+        "operationId": "BillingAccounts_Update",
+        "x-ms-long-running-operation": true,
+        "description": "The operation to update a billing account.",
+        "x-ms-examples": {
+          "UpdateBillingAccount": {
+            "$ref": "./examples/UpdateBillingAccount.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/billingAccountNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/BillingAccount"
```

Comment:
Patch typically doesn't require the same input as Put. You should create a new type definition which contains only the 'patchable' properties of a BillingAccount, the convention we typically use would name this new definition 'BillingAccountUpdateParameters'. 

Take a look at the follow swagger for an example, search for StorageAccountUpdateParameters.
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/storage/resource-manager/Microsoft.Storage/stable/2018-11-01/storage.json

For more info on how Patch should work:
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#patch-resource

Reply:
updated to use updateparameters. Currently we only allow update of billing account address.